DEPENDENCIES.md

```diff
@@ -34,6 +34,7 @@ following Free and Open Source software:
     github.com/datawire/envconfig                                                v0.0.0-20221012222025-09524dc7d59b        Apache License 2.0
     github.com/datawire/go-ftpserver                                             v0.1.1                                    Apache License 2.0
     github.com/datawire/go-fuseftp/rpc                                           v0.2.0                                    Apache License 2.0
+    github.com/datawire/k8sapi                                                   v0.1.1-0.20221102000402-bab67787b913      Apache License 2.0
```

Comment:
I like this, but shouldn't it also mean that we remove the `pkg/k8sutil` directory altogether? I mean, what's the point in having two incarnations?

Also, it seems a lot of the `watcher` functionality has generalized and moved into that package now, which means that we have multiple copies of things like `Subscribe` (copied from `pkg/client/userd/trafficmgr/workloads.go`. I think when such things are generalized, the original should be removed.

That said, I don't think this is related to the shared login, so perhaps you could create a separate PR that deals with all the "swap k8sapi" stuff?

Reply:
Yeah, I can do that in another PR in the near future